test/cmd/builds.sh

```diff
@@ -97,7 +97,7 @@ os::cmd::expect_success_and_text 'oc start-build --list-webhooks=all bc/ruby-sam
 os::cmd::expect_success_and_text 'oc start-build --list-webhooks=all ruby-sample-build' 'github'
 os::cmd::expect_success_and_text 'oc start-build --list-webhooks=github ruby-sample-build' 'secret101'
 os::cmd::expect_failure 'oc start-build --list-webhooks=blah'
-os::cmd::expect_success "oc start-build --from-webhook='$(oc start-build --list-webhooks='generic' ruby-sample-build --api-version=v1 | head -n 1)'"
+os::cmd::expect_success_and_text "oc start-build --from-webhook='$(oc start-build --list-webhooks='generic' ruby-sample-build --api-version=v1 | head -n 1)'" "build \"ruby-sample-build-[0-9]\" started"
```

Comment:
@oatmealraisin i'm still looking for a test that validates the behavior when the build can't be started because it references an invalid/unresolved imagestream.